src/mem/Pool.h

```diff
@@ -352,15 +325,13 @@ extern void memPoolIterateDone(MemPoolIterator ** iter);
 /**
  \ingroup MemPoolsAPI
  *
- * Fills a MemPoolGlobalStats with statistical data about overall
+ * Fills a MemPoolStats with statistical data about overall
  * usage for all pools.
  *
- * \param stats   Object to be filled with statistical data.
- *
  * \return Number of pools that have at least one object in use.
  *        Ie. number of dirty pools.
  */
-extern int memPoolGetGlobalStats(MemPoolGlobalStats * stats);
+extern size_t memPoolGetGlobalStats(MemPoolStats &);
```

Comment:
Please move this function into the Mem namespace.

Reply:
Done in [8250bc6](https://github.com/squid-cache/squid/pull/1186/commits/8250bc6eeb9ee0263caa9e5937dea7f02b5d5363)